### PR TITLE
Improve docs of embed_id

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
This is PR for improving docs of functions/links. Related issue: #2182

Modified functions/links:

- functions/connection/embed_id.py

### CODE REVIEW

File: chainer/functions/connection/embed_id.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def backward(self, inputs, grad_outputs):
 def embed_id(x, W, ignore_label=None):
     """Efficient linear function for one-hot input.
 
-    This function implements so called *word embedding*. It takes two
+    This function implements so called *word embeddings*. It takes two
```
Comment (ir5) [inline chainer/functions/connection/embed_id.py]: Isn't "word embedding" also good?

**Comment by ir5** [comment]
LGTM

File: chainer/functions/connection/embed_id.py (hunk 1 above)
Comment (keisuke-umezawa) [inline chainer/functions/connection/embed_id.py]: In the other part, the word "word embeddings" is used. So I uses the word "word embeddings"

Comment (ir5) [inline chainer/functions/connection/embed_id.py]: Understood.

**Comment by mitmul** [comment]
Jenkins, test this please

**Pull request merged**